src/System.Private.CoreLib/shared/System/Diagnostics/Contracts/ContractFailedEventArgs.cs

```diff
@@ -13,15 +14,15 @@ namespace System.Diagnostics.Contracts
     public sealed class ContractFailedEventArgs : EventArgs
     {
         private ContractFailureKind _failureKind;
-        private string _message;
-        private string _condition;
-        private Exception _originalException;
+        private string? _message;
+        private string? _condition;
+        private Exception? _originalException;
         private bool _handled;
         private bool _unwind;
 
-        internal Exception thrownDuringHandler;
+        internal Exception? thrownDuringHandler;
 
-        public ContractFailedEventArgs(ContractFailureKind failureKind, string message, string condition, Exception originalException)
+        public ContractFailedEventArgs(ContractFailureKind failureKind, string? message, string? condition, Exception originalException)
```

Comment:
Shouldn't originalException be `Exception?`?

Reply:
Yep, we even `Debug.Assert` for it to being `null`